.github/workflows/periodic-pytorch-libtorch-linux-xenial-cuda11.3-cudnn8-py3.6-gcc7.yml

```diff
@@ -12,7 +12,7 @@ on:
   workflow_dispatch:
 
 env:
-  BUILD_ENVIRONMENT: periodic-pytorch-libtorch-linux-xenial-cuda11.3-cudnn8-py3.6-gcc7
+  BUILD_ENVIRONMENT: pytorch-libtorch-linux-xenial-cuda11.3-cudnn8-py3.6-gcc7
```

Comment:
Is this intentional? If so, I feel like we shouldn't get rid of periodic, but instead get rid of pytorch.

Reply:
periodic shouldn't be part of the build environment --> e.g. regardless of whether the job is ran periodic or per commit behavior of the build environment is the same 

getting rid of pytorch- is in a separate PR